testing/btest/scripts/base/frameworks/logging/rotate-ascii-logdir-leftover-log-rotation-shadow-files.zeek

```diff
@@ -28,7 +28,7 @@ function my_rotation_postprocessor(info: Log::RotationInfo) : bool
 	return T;
 	}
 
-redef LogAscii::logdir = "./logs";
 redef LogAscii::enable_leftover_log_rotation = T;
+redef Log::default_logdir = "./logs";
```

Comment:
There's two tests that should cover this (and the below with stale .shadow) files. Seems reasonable to delete the "rotate-ascii-logdir-leftover*" tests:

rotate-default-logdir-leftover-log-rotation-shadow-files.zeek
rotate-default-logdir-leftover-log-rotation-stale-shadow-files.zeek